posthog/temporal/data_imports/pipelines/chargebee/__init__.py

```diff
@@ -0,0 +1,109 @@
+import base64
+from typing import Any, Optional
+
+import dlt
+import requests
+from dlt.sources.helpers.requests import Request, Response
+from dlt.sources.helpers.rest_client.paginators import BasePaginator
+
+from posthog.temporal.data_imports.pipelines.rest_source import (
+    RESTAPIConfig,
+    rest_api_resources,
+)
+from posthog.temporal.data_imports.pipelines.rest_source.typing import EndpointResource
+
+
+def get_resource(name: str, is_incremental: bool) -> EndpointResource:
+    resources: dict[str, EndpointResource] = {
+        "Customers": {
```

Comment:
I reckon starting with customers, subscriptions, orders, invoices, transactions would be good